docs/source/api.rst

```diff
@@ -54,7 +54,7 @@ All requests must be signed using `HTTP signatures <http://tools.ietf.org/html/d
               "id": 123,
```

Comment:
if you `git grep -i team|grep messages` in the root of this repository, does anything relevant pop up?

Reply:
<img width="972" alt="Screen Shot 2021-04-19 at 4 45 58 PM" src="https://user-images.githubusercontent.com/3481095/115316744-cd4ee780-a12e-11eb-920b-e1ee6440ce9c.png">

Reply:
aah that was so much better grep than what i was running :P

Reply:
ooo perfect! and i just caught a small bug in my grep: `git grep -i team|grep -i messages` (added a second -i). hopefully it doesn't uncover anything!

Reply:
The new grep was showing changes from these files:
```
docs/source/bots.rst
orchestra/frontend/build/static/js/2.564c31ea.chunk.js.map
```